crates/net/network/src/network.rs

```diff
@@ -108,7 +109,11 @@ impl<N: NetworkPrimitives> NetworkHandle<N> {
 
     /// Update the status of the node.
     pub fn update_status(&self, head: Head) {
-        self.send_message(NetworkHandleMessage::StatusUpdate { head });
+        let merge_block = ChainSpec::default().paris_block().unwrap_or(0);
+        let earliest = if head.number >= merge_block { U64::from(merge_block) } else { U64::ZERO };
+        let latest = U64::from(head.number);
+
+        self.send_message(NetworkHandleMessage::StatusUpdate { head, earliest, latest });
```

Comment:
after looking at this, I think we can keep this unchanged for now, if we ignore the case for now where the earliest block can change, so we dont actually need to change this